packages/components/src/theme/types.ts

```diff
@@ -0,0 +1,9 @@
+/**
+ * External dependencies
+ */
+import type { ReactNode, CSSProperties } from 'react';
+
+export type ThemeProps = {
+	accent?: CSSProperties[ 'color' ];
```

Comment:
We might want to document that some otherwise "valid" values cannot be parsed in this context, like `currentColor` or `var(--some-css-var)`.

Doesn't have to be in this PR, but we can also check at runtime with [`.isValid()`](https://github.com/omgovich/colord#color-analysis) and throw a dev warning.

Reply:
Added a note in the README and validation via the `isValid` function (including unit tests) in https://github.com/WordPress/gutenberg/pull/44668/commits/0ac7a24ae0164ec1c2d1c199c2668a60602a8d84

A few additional notes:

- writing tests for `isValid` actually flagged a bug in the code: we were not supporting named colors! I've added support by adding the `named` plugin
- I've also added the `a11y` plugin while I was at it, since we will need it for contrast checking (it's already used in other components, so it shouldn't really make a difference in terms of bundle size)
- I changed the type to `string`, since the `CSSProperties[ 'color' ]` was including a bunch of values that we don't support. But let me know if you'd rather revert to `CSSProperties[ 'color' ]`!

Reply:
> writing tests for `isValid` actually flagged a bug in the code

Wonderful 😄

Looks great! I think we're good to merge. Maybe copy & paste the readme stuff into JSDoc so we get it in Storybook and IntelliSense.